app/controllers/ops_controller/diagnostics.rb

```diff
@@ -152,17 +152,15 @@ def log_depot_edit
   def fetch_log
     assert_privileges("fetch_log")
     disable_client_cache
-    send_data(Vmdb::Loggers.contents($log, nil),
-              :filename => "evm.log")
```

Comment:
It's weird that the original code had both the logger object _and_ the filename...can't see why you would need both, particular when filename can be derived from the log object.

Reply:
Yeah i wasn't sure how `send_data` was using `filename` (assumed it was for display purposes?) so didn't want to mess with it for now.  Maybe we can simply delete it though will need to investigate